components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -531,6 +552,7 @@ private UserStoreDTO createUserStoreDTO(UserStoreReq userStoreReq) {
                                                                    String requiredAttributes) {
 
         List<UserStoreListResponse> userStoreListResponseToAdd = new ArrayList<>();
+        Map<String, Boolean> userStoreManagersType = UserStoreManagerRegistry.getUserStoreManagersType();
```

Comment:
Any specific reason to use the `Boolean` object type here? Can't we use primitive (`boolean`)?

Reply:
We cannot use primitive types to declare Map objects. Use of `Map<String, boolean> map` is not allowed in java.